test/generators/add-web-assets/index.test.js

```diff
@@ -36,15 +36,15 @@ afterAll(() => {
 })
 
 const expectedDefaultGenerator = expect.stringContaining(n('exc-react/index.js'))
-// const expectedPromptChoices = [expect.objectContaining({
-//   type: 'list',
-//   name: 'webAssetsGenerator',
-//   choices: [
-//     { name: 'Exc Shell React', value: expect.stringContaining(n('exc-react/index.js')) },
-//     { name: 'Raw HTML/JS', value: expect.stringContaining(n('raw/index.js')) }
-//   ],
-//   validate: utils.atLeastOne
-// })]
+const expectedPromptChoices = [expect.objectContaining({
+  type: 'list',
+  name: 'webAssetsGenerator',
+  choices: [
+    { name: 'Adobe Experience Cloud Shell - React', value: expect.stringContaining(n('exc-react/index.js')) },
+    { name: 'Raw HTML/JS', value: expect.stringContaining(n('raw/index.js')) }
```

Comment:
Raw HTML/JS UI

(Please note that this UI will also be able to deploy within Adobe Experience Cloud, so it should also integrate with the ExC Shell client-side API.